ethereum-network-partition-test/main_test.go

```diff
@@ -9,39 +9,79 @@ import (
 	"github.com/kurtosis-tech/kurtosis-sdk/api/golang/core/lib/services"
 	"github.com/kurtosis-tech/kurtosis-sdk/api/golang/engine/lib/kurtosis_context"
 	"github.com/kurtosis-tech/stacktrace"
+	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/require"
 	"math/big"
+	"sort"
+	"strings"
 	"sync"
 	"testing"
 	"time"
 )
 
 /*
-This test will start a three-node Ethereum network, partition it, and then heal the partition to demonstrate
-Ethereum-forking behaviour in Kurtosis.
+This example will:
+1. Start an Ethereum network with `numParticipants` nodes
+2. Wait for all the nodes to be synced
+3. Partition the network into 2. Half of the nodes will be in partition 1, half will be in partition 2
+4. Wait for the block production to diverge in each partition
+5. Heal the partition and wait for all nodes to get back in sync
+
+This test demonstrate Ethereum-forking behaviour in Kurtosis.
 */
 
 const (
+	logLevel = logrus.InfoLevel
+
 	testName              = "go-network-partitioning"
 	isPartitioningEnabled = true
 
+	nodeInfoPrefix = "NODES STATUS -- |"
+
 	ethModuleId    = "eth-module"
-	ethModuleImage = "kurtosistech/eth2-merge-kurtosis-module:7.0.0"
-	moduleParams   = `{
+	ethModuleImage = "kurtosistech/eth2-merge-kurtosis-module:gbouv_add-fee-recipient"
```

Comment:
TODO: update this once https://github.com/kurtosis-tech/eth2-merge-kurtosis-module/pull/143 is merged an released